### ISSUE TITLE
Anti-aliasing isn't applied to "SHOW COMMENTS" button.

### ISSUE THREAD

**Issue by 1natsu172** [dideler/toggle-youtube-comments#25]
Since anti-aliasing isn't applied, the font-weight of the "SHOW COMMENTS" button seems to be different from the original one.

![](https://i.gyazo.com/0ef4ec64a4bfc8d8dc85f52b784b3c81.png)

We can understand it by browsing in Youtube's dark mode.

![](https://i.gyazo.com/64f6f82510361b1095b70809da1c7993.png)

So, i think that it is all right with this modify method.

```
.fake-paper-button {
    -webkit-font-smoothing: antialiased;
}
```

Should we fix this problem with issue #2 ? Because it is a button problem.

**Comment by dideler** [comment]
Good spot. I think this can be a separate fix, so it's not blocked by #2.

**Comment by 1natsu172** [comment]
OK, let's deal with these issues separately.

**Comment by 1natsu172** [comment]
Fixed this issue by #30 🔫
